airflow/example_dags/example_subdag_operator.py

```diff
@@ -0,0 +1,55 @@
+from datetime import datetime
+
+from airflow.models import DAG
+from airflow.operators import DummyOperator, SubDagOperator
+
+from airflow.example_dags.subdags import subdag
+
+
+args = {
+    'owner': 'airflow',
+    'start_date': datetime(2016, 1, 1),
+}
+
+dag = DAG(
+    dag_id='example_subdag_operator',
```

Comment:
Make this (`example_subdag_operator`) a constant and pass the constant into the subdag() function calls below as another parameter.